packages/react-app/pages/admin/newBatches.jsx

```diff
@@ -0,0 +1,373 @@
+import React, { useEffect, useState, useMemo } from "react";
+import axios from "axios";
+import moment from "moment";
+import NextLink from "next/link";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Center,
+  Container,
+  Link,
+  Text,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  chakra,
+  Flex,
+  Select,
+  Tooltip,
+  HStack,
+  Input,
+  InputRightElement,
+  InputGroup,
+  useClipboard,
+} from "@chakra-ui/react";
+import { useTable, usePagination, useSortBy, useFilters } from "react-table";
+import { CopyIcon, SearchIcon, TriangleDownIcon, TriangleUpIcon } from "@chakra-ui/icons";
+import BuilderListSkeleton from "../../components/skeletons/BuilderListSkeleton";
+import DateWithTooltip from "../../components/DateWithTooltip";
+import SocialLink from "../../components/SocialLink";
+import Address from "../../components/Address";
+import { bySocialWeight } from "../../data/socials";
+import { USER_ROLES } from "../../helpers/constants";
+import StreamTableCell from "../../components/StreamTableCell";
+import MetaSeo from "../../components/MetaSeo";
+import BuilderFlags from "../../components/builder/BuilderFlags";
+import useCustomColorModes from "../../hooks/useCustomColorModes";
+import BatchColumnFilter from "../../components/BatchColumnFilter";
+
+const serverPath = "/builders";
```

Comment:
Question: why is it useful to have all the builders on this page?

I thought it was meant only for batch people. And we have and endpoint to just fetch people with a batch.

Reply:
I initially thought it would be nice to have a page where you can filter through all the builders, but you are right, since there's already a builders page available for everyone, it might not be necessary.  

However, once we would implement batch editing (as you suggested below), and need to add builders to batches, it could become important to show all builders so we can easily select the latest ones and assign them to batches.

What do you think? But I can also directly use `/builders/batches` and only batch people.

Reply:
Both things make sense! But I was expecting to see only people from the batches.

> However, once we would implement batch editing (as you suggested below), and need to add builders to batches, it could become important to show all builders so we can easily select the latest ones and assign them to batches.

The next PR (after this) might be the automation of adding people as candidates, right? (they sign a message to get the TG link and get auto-tagged as candidates)... so this won't be needed.

In any case, you are the one who is going to use that page the most, so whatever makes more sense to you!

Reply:
Okay, I will remove it.